src/mlpack/methods/rann/krann_main.cpp

```diff
@@ -204,48 +204,48 @@ static void mlpackMain()
     rann->RandomBasis() = randomBasis;
 
     Log::Info << "Using reference data from "
-        << CLI::GetPrintableParam<arma::mat>("reference") << "." << endl;
-    arma::mat referenceSet = std::move(CLI::GetParam<arma::mat>("reference"));
+        << IO::GetPrintableParam<arma::mat>("reference") << "." << endl;
+    arma::mat referenceSet = std::move(IO::GetParam<arma::mat>("reference"));
 
     rann->BuildModel(std::move(referenceSet), size_t(lsInt), naive, singleMode);
   }
   else
   {
     // Load the model from file.
-    rann = CLI::GetParam<RANNModel*>("input_model");
+    rann = IO::GetParam<RANNModel*>("input_model");
 
     Log::Info << "Using rank-approximate kNN model from '"
-        << CLI::GetPrintableParam<RANNModel*>("input_model") << "' (trained on "
+        << IO::GetPrintableParam<RANNModel*>("input_model") << "' (trained on "
         << rann->Dataset().n_rows << "x" << rann->Dataset().n_cols
         << " dataset)." << endl;
 
     // Adjust singleMode and naive if necessary.
-    rann->SingleMode() = CLI::HasParam("single_mode");
-    rann->Naive() = CLI::HasParam("naive");
+    rann->SingleMode() = IO::HasParam("single_mode");
+    rann->Naive() = IO::HasParam("naive");
     rann->LeafSize() = size_t(lsInt);
   }
 
   // Apply the parameters for search.
-  if (CLI::HasParam("tau"))
-    rann->Tau() = CLI::GetParam<double>("tau");
-  if (CLI::HasParam("alpha"))
-    rann->Alpha() = CLI::GetParam<double>("alpha");
-  if (CLI::HasParam("single_sample_limit"))
-    rann->SingleSampleLimit() = CLI::GetParam<int>("single_sample_limit");
-  rann->SampleAtLeaves() = CLI::HasParam("sample_at_leaves");
-  rann->FirstLeafExact() = CLI::HasParam("sample_at_leaves");
+  if (IO::HasParam("tau"))
+    rann->Tau() = IO::GetParam<double>("tau");
+  if (IO::HasParam("alpha"))
+    rann->Alpha() = IO::GetParam<double>("alpha");
+  if (IO::HasParam("single_sample_limit"))
+    rann->SingleSampleLimit() = IO::GetParam<double>("single_sample_limit");
```

Comment:
```suggestion
    rann->SingleSampleLimit() = IO::GetParam<int>("single_sample_limit");
```
@shrit should be an easy fix, I think it was just a bad merge. :) #2442 corrected the type of `single_sample_limit`, but my guess is that git just overwrote that change when you merged the branch in. :+1:

Reply:
I agree I had bad merged since it did not notice the change from double to int